makefile

```diff
@@ -377,6 +377,7 @@ MFEM_EXT_LIBS  ?= $(ALL_LIBS) $(LDFLAGS) $(LDFLAGS_INTERNAL)
 MFEM_LIBS      ?= $(if $(shared),$(BUILD_RPATH)) -L@MFEM_LIB_DIR@ -lmfem\
    @MFEM_EXT_LIBS@
 MFEM_LIB_FILE  ?= @MFEM_LIB_DIR@/libmfem.$(if $(shared),$(SO_VER),a)
+MFEM_LIBE_FILE ?= @MFEM_LIB_DIR@/libmfem-extras.$(if $(shared),$(SO_VER),a)
```

Comment:
A new library file? Why is the code in it not put in `libmfem.*`? This is a big change that needs to be discussed if it is really needed.

Reply:
Sorry. I believe this is an artifact from a very old change which was reverted in favor of `miniapps/common`. I'll clean it up.